src/device-registry/utils/translate.js

```diff
@@ -0,0 +1,50 @@
+const constants = require("@config/constants");
+const log4js = require("log4js");
+const logger = log4js.getLogger(
```

Comment:
Hi @BenjaminSsempala ....in an effort to minimise the number of "new files", could you add these as "helper functions" above the `create-health-tips `module?
WHY? I am just worried of an exponential number of new files which can consequentially make our microservice intimidating to a new user. Additionally, it can also make maintenance tricky. One major benefit of the concept of "reusing" is that it forces us to appreciate what is already existing in our codebases instead of just piling new files for every new functionality.

 Suggested refactoring above is more like below: 

const translate = (healthTips, targetLanguage) => { 
// function body
}
const translateText = (text, target) => {
// function body
}


const createHealthTips = {
  list: async (request) => { },
  fn2: async (request) => {},
 ......
}
module.exports = createHealthTips

Reply:
Thanks @Baalmart 
I'd added this as a separate util file because the same util will also be utilised by the KYA lessons and quiz or any other API output that needs translation from the mobile app.
Would you recommend that each Util has its own translate helper function?

Reply:
Okay, thanks for the clarification @BenjaminSsempala !

Reply:
I think we can maintain the current structure. Explanation makes sense, as long as we do not create new file for every new function, we shall keep the file size manageable. @BenjaminSsempala